core/Db/Schema/Mysql.php

```diff
@@ -334,6 +334,24 @@ public function getTablesCreateSql()
                                       ) $tableOptions
             ",
 
+            'archiving_metrics'     => "CREATE TABLE {$prefixTables}archiving_metrics (
+                                      metadataid BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                                      idarchive BIGINT UNSIGNED NULL,
+                                      idsite INTEGER UNSIGNED NOT NULL,
+                                      segment VARCHAR(255) NULL,
+                                      date1 DATE NOT NULL,
+                                      date2 DATE NOT NULL,
+                                      period VARCHAR(10) NOT NULL,
+                                      ts_started DATETIME NOT NULL,
+                                      ts_finished DATETIME NOT NULL,
+                                      total_time BIGINT UNSIGNED NOT NULL,
+                                      total_time_exclusive BIGINT UNSIGNED NOT NULL,
+                                        PRIMARY KEY(metadataid),
+                                        INDEX idx_archiving_metrics_idarchive(idarchive),
+                                        INDEX idx_archiving_metrics_site_date_period(idsite, date1, period)
```

Comment:
```suggestion
                                        INDEX index_idarchive(idarchive),
                                        INDEX index_idsite_date_period(idsite, date1, period)
```

To keep the names somewhat consistent with other indices.